### DISCUSSION TITLE
how to build react on development mode?

### DISCUSSION THREAD

**Discussion by ITSWYoo** [vercel/next.js#17955] [Help]
i have 5 env `local, development, alpha, beta(stage), production`
next dev is used to only local 
but, i want to build as development mode or alpha for not minify, uglify, and to use browser react developer tools.
but `next build` is always run as production.
help me plz : )

**Comment by alanqchen** [comment]
Using just `next` or `npm run dev` (if you have the script defined) should run the development mode.

**Reply by ITSWYoo** [reply]
i want to build (not hmr, not minify, uglify, and to use browser react developer tools)
